### DISCUSSION TITLE
Qiitaニュースのメールタイトル記事が本文のどこにあるのかわかりにくい？ あるいはメールの構成について

### DISCUSSION THREAD

**Discussion by nibuno** [increments/qiita-discussions#212] [要望・アイデア・不具合]
こんにちは！Qiitaのディスカッションがあると知ったので投稿してみます。
もっとQiitaのメール体験を向上させることが出来る余地があるかな？と言うのが投稿の意図です。
「メール」で検索して同様の投稿が無さそうと思いましたが、別にありましたら誘導してください。

毎週1回メール配信してくれているQiitaニュースについて。
個人的な意見なのですが、メールタイトルの記事で興味を持ってメールを開いてみたら
どこに記事があるのか分からなくて少し戸惑ってしまいました。

以下のスクリーンショットで言うと、
タイトル：Magicodeという簡単に稼げるエンジニア向けブログサービスを作ったので使って欲しい
の記事が
**先週LGTMが多かった投稿ベスト20** の7位ですね。結構スクロールしてタイトルを覚えつつ探さないと見つからないように思いました。

<img width="1063" alt="スクリーンショット 2022-04-29 17 06 47" src="https://user-images.githubusercontent.com/41376154/165907014-c4298d4b-a907-48c5-98a4-72a4542db2fa.png">
<img width="1063" alt="スクリーンショット 2022-04-29 17 06 57" src="https://user-images.githubusercontent.com/41376154/165907026-1084f834-f825-4abc-ad4c-17634dc45b09.png">

以下のような案が考えられましたが、どれも一長一短（かつ諸事情があるのかな）と思っています。
具体的な解決策ではない、かつ素人の考えで恐縮ですが、ディスカッションできると嬉しいです。

**タイトルを変更する**
- ex: Qiitaニュース Vol xx, Qiitaニュース YYYY/MM/DD
　- メリット： 違和感が減る気がする
　- デメリット：せっかくのメールタイトルでメールを開こうとする訴求ポイントが失われる

**本文の構成を変更する**
-  ex: 今週のおすすめ記事はこちら！「xxxx」その後にQiita運営からのお知らせ などを続ける
　- メリット：タイトルと本文の導線が良くなる
　- デメリット：お知らせを最初に見ていた人たちが見なくなる？ PR記事などを読まなくなる可能性もある？（PR記事などは企業さんとの取り決めなどあったら構成を変更することでややこしくならないかな？と言う点が気になっています）

**メールを分ける**
- ex: Qiita運営からのお知らせ, Qiita Zine更新情報, 先週LGTMが多かった投稿ベスト20 など別々のメールにする
　- メリット：タイトルと本文の内容がマッチする
　- デメリット：内部の実装、運用が今より複雑化する（？）, ユーザーへの同意を求めたりする機会が増える（？） 主に社内的な負担が大きくなりそう

と、こんな感じのことを考えましたがいかがでしょうか？

**Comment by JunichiIto** [comment]
> 個人的な意見なのですが、メールタイトルの記事で興味を持ってメールを開いてみたら
どこに記事があるのか分からなくて少し戸惑ってしまいました。

これ、僕も経験あります。以前からちょっと気になってました😅

**Comment by tanaka-asami415** [comment]
@Yumihiki 　@JunichiIto 
お二人ともありがとうございます🙏

> PR記事などは企業さんとの取り決めなどあったら構成を変更することで
ややこしくならないかな？と言う点が気になっています

まさに、仰る通りでして、一旦社内にもちかえって
クライアントとの兼ね合いも見て検討させていただきます🙏

**Reply by nibuno** [reply]
面倒なことを言ってしまってすみません🙏
社内でも前向きな検討いただけると嬉しいです！🙇‍♂️
